src/extract.py

```diff
@@ -8,18 +9,23 @@
 from common import get_log
 
 log = get_log(__file__)
-READERS = {".xlsx": pd.read_excel, ".xls": pd.read_excel, ".csv": pd.read_csv}
 
 
 def read_to_dict(infile):
-    path_info = pathlib.Path(infile)
-    reader = READERS[path_info.suffix]
-    records = reader(infile).replace({np.nan: None}).to_dict(orient="records")
-    return records
+    payload = infile.read(2048)
```

Comment:
let's think of a better function name since read_to_dict doesn't return a dict so its confusing. Also what are we reading? Does this function belong in the extract step?

Reply:
That's a good point. I've renamed it to `records_from_file` and moved it to `common.py` since it's now used by extract and the flask app.

It returns a list of json records (dicts) , I've updated the docstring and expanded the test to cover both input file and stream.